modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro

```diff
@@ -675,6 +675,20 @@ definition {
 			value1 = "${fieldValue}");
 	}
 
+	macro setFieldPredefinedImage {
+		Form.gotoAdvancedTab();
+
+		Click(locator1 = "FormFields#UPLOAD_PREDEFINED_VALUE");
+
+		ItemSelector.uploadFile(
+			navTab = "Documents and Media",
+			uploadFileName = "${uploadFileName}");
+
+		AssertElementPresent(
+			key_imageName = "${uploadFileName}",
+			locator1 = "FormViewBuilder#SIDEBAR_ADVANCED_PREDEFINED_IMAGE");
+	}
+
```

Comment:
Hi @leticiachiu, the App Builder folder has been deleted from Master. Can you move this macro to `FormFields.macro` file and rename it to `setImagePredefinedValue`? Thanks!